### ISSUE TITLE
Feature request: optionally allow primes to persist through normalization on double turns. For example: U2' => U2'

### ISSUE THREAD

**Issue by bnat1** [Zahajki/cube-notation-normalizer#1]
I understand that in "standard" notation, double moves aren't usually inverted. But for execution, it can be helpful to distinguish double turns from prime double turns. It would be great if there was an option, something like { allowDoublePrime: true/false }, with default false, that allows for normalizing with execution. 

I'm going to fork this project and implement this change. I'll comment here / make a pr when the change is made in case you want to include this functionality in the project.

**Comment by Zahajki** [comment]
I'm not sure about what you really want, anyway, notations like `U2'` are already supported (without any options). There is no restriction about the order of primes and numbers.
```javascript
normalize("U2'");
// => "U2"
normalize("U'2");
// => "U2"
```
For more detail, see [syntax definition](https://github.com/Zahajki/cube-notation-normalizer/blob/235b7a393118ca0d4f4e3e8c76514e717b84d52b/notation.pegjs#L28) and [test case](https://github.com/Zahajki/cube-notation-normalizer/blob/235b7a393118ca0d4f4e3e8c76514e717b84d52b/tests/index.test.js#L41).

**Comment by bnat1** [comment]
Right, what I meant, was it would be nice to optionally allow the prime to persist after normalization, like this:
`
normalize("U2'", { doublePrimeMoves: true });
// => "U2'"
`
The reason being, imagine that I want to preserve how an algorithm is executed. Currently, the detail of the turn being prime would be lost through normalization.
